ignite/base/events_driven.py

```diff
@@ -304,14 +310,22 @@ def __setattr__(self, attr: str, value: Any) -> None:
 
         super().__setattr__(attr, value)
 
-    def update_mapping(self, event_to_attr: Mapping[Any, str]) -> None:
-        """Looks for the attribute and check if not existed to add it.
+    def update_mapping(self, event_to_attr: Mapping["Events", str]) -> None:
```

Comment:
We can change the signature and replace `event_to_attr` by `attr_to_events`

Reply:
That will require refactoring in the tests, also another another thought I wanted to keep both options for the user, either to give `attr_to_events` in the constructor or `event_to_attr` in `update_mapping`, what do you think?